### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/exiv2/commit/70a7c49f45456c86e511644b3108242566a598a2#commitcomment-65911822

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/exiv2/commit/37f930d9eb667c61a98c4307011a509c329cb6bb

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I intend to annotate this PR shortly. I'm happy to drop changes to files, drop specific terms, or split things as requested, although this may take some time.

### CODE REVIEW

**Review by sridharb1 — DISMISSED** [review]
Thank you, looks good

File: exiv2.md
Comment (jsoref) [inline exiv2.md:1012]: This is probably the biggest stretch as a spelling fix. Specifically, `UCT` isn't a thing.

There are 18x as many hits for `"Coordinated Universal Time"` than `"Universal Coordinated Time"`, and the top hits for the latter show the former.

File: include/exiv2/types.hpp
Comment (jsoref) [inline include/exiv2/types.hpp:105]: Sometimes I run into projects that are trying to do something special here. But I think this is right.

File: po/exiv2.pot
Comment (jsoref) [inline po/exiv2.pot:13404]: I've mostly ignored the `.pot` file (and probably should have ignored it entirely).

File: releasenotes/releasenotes.txt
Comment (jsoref) [inline releasenotes/releasenotes.txt:28]: Interesting stray near duplicate corruption.

File: releasenotes/releasenotes.txt
Comment (jsoref) [inline releasenotes/releasenotes.txt:32]: Perhaps en-GB spelling would be better...

File: samples/conntest.cpp
Comment (jsoref) [inline samples/conntest.cpp:76]: or `happens`?

File: src/canonmn_int.cpp
Comment (jsoref) [inline src/canonmn_int.cpp:744]: I'm fairly certain these are user facing localization elements...

File: src/casiomn_int.cpp
Comment (jsoref) [inline src/casiomn_int.cpp:374]: I suppose I could add a space for each of the other rows

File: src/crwimage_int.cpp
Comment (jsoref) [inline src/crwimage_int.cpp:1106]: This is probably the biggest change in the PR. I'm not sure about it.

File: src/nikonmn_int.cpp
Comment (jsoref) [inline src/nikonmn_int.cpp:64]: moderately confident this is right

File: src/nikonmn_int.cpp
Comment (jsoref) [inline src/nikonmn_int.cpp:1681]: This could be wrong if it's `focus` + `used` instead of `focussed`.

File: src/panasonicmn_int.cpp
Comment (jsoref) [inline src/panasonicmn_int.cpp:604]: I had to look this one up...

File: xmpsdk/include/TXMPUtils.hpp
Comment (jsoref) [inline xmpsdk/include/TXMPUtils.hpp:772]: I don't see any particular reason to shorten this token, but I'm happy to drop it (or any other change(s)) if requested.

File: xmpsdk/include/client-glue/TXMPMeta.incl_cpp
Comment (jsoref) [inline xmpsdk/include/client-glue/TXMPMeta.incl_cpp:94]: Here and below are places where apparently the bottom `=`/`-` should be as long as the text...

File: xmpsdk/src/XMPUtils-FileInfo.cpp
Comment (jsoref) [inline xmpsdk/src/XMPUtils-FileInfo.cpp:214]: `-`....

**Review by jsoref — COMMENTED** [review]
The vast majority of the corrections were automatically suggested by Google Sheets (especially any preference for en-US over en-GB).

But all fault is mine.

File: doc/ChangeLog
Comment (piponazo) [reply doc/ChangeLog:326]: It is fine for me to fix typos in the changlog too 👍

File: doc/ChangeLog
Comment (piponazo) [reply doc/ChangeLog:646]: I would wait for @clanmills feedback on this matter. He has been the main maintainer of the project for many years, and I am sure he will have something to say about this 😸.

I do not have a strong preference for one or another.

File: xmpsdk/include/TXMPUtils.hpp
Comment (piponazo) [reply xmpsdk/include/TXMPUtils.hpp:772]: I would vote to drop all the changes done inside the **xmpsdk** folder. This is a 3rd party dependency in the project that was included in our codebase a long time ago, but we try to not touch these files too much.

**Review by piponazo — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution! It is really appreciated.

I would also love to see (probably in a separated PR) the inclusion on the action in the project. Most of the collaborators are non native English-speakers, and it will probably find us to help typos eventually.

I made few comments here and there about things we could drop.

**Comment by sridharb1** [comment]
Yes, Luis, I also thought about the focus used making sense, but I also thought that focused also makes sense, as in, if it is focused, then focus is used.




On Thursday, February 3, 2022, 13:35, Luis Díaz Más ***@***.***> wrote:




@piponazo requested changes on this pull request.

Thanks a lot for the contribution! It is really appreciated.

I would also love to see (probably in a separated PR) the inclusion on the action in the project. Most of the collaborators are non native English-speakers, and it will probably find us to help typos eventually.

I made few comments here and there about things we could drop.

In po/exiv2.pot:
> -"References to resources that were incorporated, byinclusion or reference, "
+"References to resources that were incorporated, by inclusion or reference, "

I think it is fine, since you also fixed the typo in the properties.cpp file.

In src/crwimage_int.cpp:
> -            // Try the undecoded tag
+            // Try the raw tag

Right ... I am not very sure about what would be the right word to use there. I would vote to drop this change.

In src/nikonmn_int.cpp:
> -            unsigned focusused = (value.toLong(2) << 8) + value.toLong(3);
+            unsigned focussed = (value.toLong(2) << 8) + value.toLong(3);

Hehe interesting ... I think this should be dropped because it problably means "FocusUsed". As you can see between lines 1721-1733, we are trying to determine if a focus point was used (and which one) or not. This correspond to AFPointsInFocus:
https://www.exiftool.org/TagNames/Nikon.html#AFInfo

In xmpsdk/include/TXMPUtils.hpp:
> -									 tStringObj *                 catedStr );
+									 tStringObj *                 concatenatedStr );

I would vote to drop all the changes done inside the xmpsdk folder. This is a 3rd party dependency in the project that was included in our codebase a long time ago, but we try to not touch these files too much.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by clanmills** [comment]
@sridharb1 Thank You for saying _"I would vote to drop all the changes done inside the xmpsdk folder. "_.  I agree and you've explained the reason.

@jsoref This is a really good contribution.  I'm a native english speaker (Scottish) and I'm dyslexic (also Scottish).  So all the typos in README.md and elsewhere are mine.  Thank You for working on this.  I prefer en-US because it's more conventional for Open-Source. _(I'm also an US Citizen)_.

When you've finished on this, I'd appreciate similar treatment being given to my book:  https://github.com/Exiv2/team.exiv2.org

File: README.md
Comment (kmilos) [reply README.md:9]: This is fine, but note this will lead to a completely new workflow created [here](https://github.com/Exiv2/exiv2/actions) (with the same name though), and [the old runs will have to be deleted via the API](https://github.com/Exiv2/exiv2/pull/2049#issuecomment-1006710079).

File: include/exiv2/datasets.hpp
Comment (kmilos) [inline include/exiv2/datasets.hpp:190]: Photoshop should also be capitalized (as a brand name)? See also line 193 below (and look for other instances)?

**Review by piponazo — APPROVED** [review]

**Pull request merged**
